docs/package.json

```diff
@@ -9,6 +9,7 @@
         "build": "astro check && astro build",
         "preview": "astro preview",
         "astro": "astro",
+        "check-format": "prettier --ignore-path \"../.gitignore\" --ignore-path .gitignore  --check \"./**/*.{ts,tsx,json,astro,md,mdx,mjs,cjs}\"",
         "format": "prettier --ignore-path \"../.gitignore\" --write \"./**/*.{ts,tsx,json,astro,md,mdx,mjs,cjs}\""
```

Comment:
```suggestion
        "format": "prettier --ignore-path 
        "../.gitignore\" --ignore-path .gitignore  --write \"./**/*.{ts,tsx,json,astro,md,mdx,mjs,cjs}\""
```

Reply:
I think we should also add the `.gitignore` here? Or do you want to format things that are ignored by git with `format`?